data/api/server-server/space_hierarchy.yaml

```
@@ -0,0 +1,225 @@
+                            - type: object
+                              properties:
+                                origin_server_ts:
+                                  type: number
+                                  format: int64
+                                  description: The `origin_server_ts` for the event.
+                              required: [origin_server_ts]
+                    required: [room_type, allowed_room_ids, children_state]
+              children:
+                description: |-
+                  A summary of the space's children. Rooms which the requesting server cannot peek/join will
+                  be excluded.
+                allOf:
+                  - $ref: "../client-server/definitions/public_rooms_chunk.yaml"
+                  - type: object
+                    properties:
+                      room_type:
+                        type: string
+                        description: |-
+                          The `type` of room (from [`m.room.create`](/client-server-api/#mroomcreate)), if any.
+                      allowed_room_ids:
+                        type: array
+                        items:
+                          type: string
+                        description: |-
+                          If the room is a [restricted room](#restricted-rooms), these are the room IDs which
+                          are specified by the join rules. Empty otherwise.
+                      children_state:
+                        type: array
+                        description: |-
+                          The [`m.space.child`](/client-server-api/#mspacechild) events of the space-room, represented
+                          as [Stripped State Events](/client-server-api/#stripped-state) with an added
+                          `origin_server_ts` key.
+
+                          If the room is not a space-room, this should be empty.
+                        items:
+                          allOf:
+                            - $ref: "../../event-schemas/schema/core-event-schema/stripped_state.yaml"
+                            - type: object
+                              properties:
+                                origin_server_ts:
+                                  type: number
+                                  format: int64
+                                  description: The `origin_server_ts` for the event.
+                              required: [origin_server_ts]
+                    required: [room_type, allowed_room_ids, children_state]
+              inaccessible_children:
+                type: array
+                items:
+                  type: string
+                description: |-
+                  The list of room IDs the responding server cannot provide details on. Rooms which the requesting
+                  server doesn't have a viable way to peek/join will be outright excluded from the response instead.
```

I think this is swapped:

```suggestion
                  The list of room IDs the requesting server doesn't have a viable way to peek/join. Rooms which
                  the responding server cannot provide details on will be outright excluded from the response instead.
```

---

applied manually